cpp/modmesh/pilot/R3DWidget.cpp

```diff
@@ -84,8 +84,34 @@ void R3DWidget::updateWorld(std::shared_ptr<WorldFp64> const & world)
             child->deleteLater();
         }
     }
-    new RVertices(world, m_scene);
+
     new RLines(world, m_scene);
+    new RVertices(world, m_scene);
```

Comment:
Create a QObject and set parent. Discard using raw pointers or smart pointers to take QObject's ownership.